lib/screens/release_health/release_health.dart

```diff
@@ -31,90 +20,101 @@ class ReleaseHealth extends StatefulWidget {
 }
 
 class _ReleaseHealthState extends State<ReleaseHealth> {
-  Future<List<Release>> futureReleases;
   int _index = 0;
 
+  var refreshKey = GlobalKey<RefreshIndicatorState>();
```

Comment:
```suggestion
  var _refreshKey = GlobalKey<RefreshIndicatorState>();
```

I'd keep it private if possible, if it needs to be public, what's about a read-only getter?